.github/scripts/release-workflow-package-push.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```

Comment:
Looking at the raw diff (https://patch-diff.githubusercontent.com/raw/heroku/buildpacks-jvm/pull/12.diff) the file has perms 644 - might need to be updated to 744 depending on how the workflow bash runner works.

Reply:
Thank you! Good catch!